docs/interpreter/livy.md

```diff
@@ -44,6 +49,26 @@ Additional requirements for the Livy interpreter are:
     <td>1000</td>
     <td>Max number of SparkSQL result to display.</td>
   </tr>
+    <tr>
+    <td>livy.spark.driver.cores</td>
+    <td>1</td>
+    <td>Driver cores. ex) 1, 2.</td>
+  </tr>
+    <tr>
+    <td>livy.spark.driver.memory</td>
+    <td>512m</td>
+    <td>Driver memory. ex) 512m, 32g.</td>
+  </tr>
+    <tr>
+    <td>livy.spark.executor.instances</td>
+    <td>3</td>
+    <td>Executor instances. ex) 1, 4.</td>
+  </tr>
+    <tr>
+    <td>livy.spark.executor.cores</td>
+    <td>1</td>
+    <td>Max number of SparkSQL result to display.</td>
+  </tr>
```

Comment:
I think you should put these on doc as well;

    livy.spark.executor.memory
    livy.spark.dynamicAllocation.enabled
    livy.spark.dynamicAllocation.cachedExecutorIdleTimeout
    livy.spark.dynamicAllocation.minExecutors
    livy.spark.dynamicAllocation.initialExecutors
    livy.spark.dynamicAllocation.maxExecutors